src/AuthRoute.js

```diff
@@ -1,14 +1,15 @@
 import React, { Component } from 'react'
 import { Route, Redirect } from 'react-router-dom'
 import Navbar from './components/Navbar'
+import { dashboard } from './urls'
```

Comment:
this isnt in the scope of this pr right? You can leave this as how it was previously

Reply:
Ok!